lib/recurly/account_acquisition.php

```diff
@@ -22,7 +22,8 @@ public static function deleteForAccount($accountCode, $client = null) {
   }
 
   protected static function uriForAccountAcquisition($accountCode) {
-    return '/accounts/' . rawurlencode($accountCode) . '/acquisition';
+    $path =  self::_uriForResource(Recurly_Client::PATH_ACCOUNTS, rawurlencode($accountCode));
```

Comment:
Wouldn't it make more sense to give the responsibility of calling `rawurlencode` on all the component parts to the `_uriForResource` method now that we are using that?

Reply:
Totally!